coordinator/src/orderbook/trading.rs

```diff
@@ -230,6 +268,30 @@ impl Trading {
 
         Ok(order)
     }
+
+    /// Notifies the trader if a pending match is waiting for them.
+    pub async fn process_pending_match(&self, trader_id: PublicKey) -> Result<()> {
+        let mut conn = self.pool.get()?;
+        if let Some(order) =
+            orders::get_by_trader_id_and_state(&mut conn, trader_id, OrderState::Match)?
+        {
+            tracing::debug!(%trader_id, order_id=%order.id, "Notifying trader about pending match");
+
+            let matches = matches::get_matches_by_order_id(&mut conn, order.id)?;
+            let filled_with = FilledWith::try_from(matches)?;
+
+            let message = match order.order_reason {
+                OrderReason::Manual => OrderbookMsg::Match(filled_with),
+                _ => OrderbookMsg::ForceMatch { order, filled_with },
```

Comment:
This might mess with us in the future: every non-manual order will be a `ForcedMatch`. Can you say this for sure? If not, maybe better to write all combinations out so that we see if a new variant was added.

Reply:
I made it explicit, so we do not run into pitfalls when adding a new variant.